org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/BuildManager.xtend

```diff
@@ -36,13 +37,18 @@ class BuildManager {
 
     val dirtyFiles = <URI>newLinkedHashSet
     val deletedFiles = <URI>newLinkedHashSet
+    List<IResourceDescription.Delta> unreportedDeltas = newArrayList 
     
     def Buildable submit(List<URI> dirtyFiles, List<URI> deletedFiles) {
     	queue(this.dirtyFiles, deletedFiles, dirtyFiles)
         queue(this.deletedFiles, dirtyFiles, deletedFiles)
         return [cancelIndicator|internalBuild(cancelIndicator)]
     }
 
+	/** 
```

Comment:
This class is currently indented with spaces. please use spaces this for this change also. Normally tabs are preferred, but start mixing makes it worse.

Reply:
Done. Why is everybody so much about whitespace recently? ;-)